### PR TITLE
Make new post page and change post page

### PR DESCRIPTION
**Pull request by vakyl9r**

### CODE REVIEW

**Comment by vakyl9r** [comment]
@Magons what should I do and how?

**Comment by Magons** [comment]
@vakyl9r `rebase branch_name`

**Comment by vakyl9r** [comment]
@Magons thanks, that's what I thought. Just wanted to make sure I'm right.

**Comment by vakyl9r** [comment]
@Magons rebased

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -12,4 +12,4 @@
  *
  */
 @import "bootstrap";
-@import 'scaffolds';
+@import "scaffolds";
```
Comment (Magons) [inline app/assets/stylesheets/application.scss]: why do you need `"` here?

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,3 +1,8 @@
+require "application_responder"
```
Comment (Magons) [inline app/controllers/application_controller.rb]: use single quotes

File: app/controllers/comments_controller.rb [hunk 3]
```diff
@@ -1,10 +1,11 @@
 class CommentsController < ApplicationController
   before_action :set_comment, only: [:show, :edit, :update, :destroy]
-
+  respond_to :json, :html
```
Comment (Magons) [inline app/controllers/comments_controller.rb]: do you realy need `:json' here? 
don't need to write code that we don't use!

**Comment by vakyl9r** [comment]
@Magons ooooops. Sorry, I've created shitload of commits(
Still figuring out how to rebase properly, I'm getting conflicts all the time, and I don't know how to properly fix them.

**Comment by Magons** [comment]
after command `git rebase development` you get message about conflicts.
1. git diff (see some files)

```
iff --cc app/assets/stylesheets/application.scss
index 1a0cb24,839c542..0000000
--- a/app/assets/stylesheets/application.scss
+++ b/app/assets/stylesheets/application.scss
@@@ -12,4 -12,4 +12,8 @@@
   *
   */
  @import "bootstrap";
++<<<<<<< de1b162ad6b68e35f9d221b096cd4f5d5258addc
 +@import 'scaffolds';
++=======
+ @import "scaffolds";
++>>>>>>> #6 Made new post page and edit post page. Installed responders, modified controllers accordingly. Converted post pages to haml. Minor addition to stylesheets
```
1. in texteditor go in this file `/app/assets/stylesheets/application.scss` and remove extra lines
   `<<<<<<< de1b162ad6b68e35f9d221b096cd4f5d5258addc`
   `=======`
   `@import "scaffolds";`
   `>>>>>>> #6 Made new post page and edit post page. Installed responders, modified controllers accordingly. Converted post pages to haml. Minor addition to stylesheets`
2. in console `git add .`
3. `git rebase --continue` 
4. Do this cycle, when count of conflicts will be 0

**Pull request merged**